app/pages/home-for-user/recent-messages.jsx

```diff
@@ -131,24 +135,31 @@ class RecentCollectionsSection extends React.Component {
       this.setState({
         messageAuthors: authorState,
       });
-      return author.get('avatar')
-      .catch(() => {
-        return [];
-      })
-      .then((avatars) => {
-        const avatar = [].concat(avatars)[0]; // Why's this an array?
-        const avatarState = Object.assign({}, this.state.avatars);
-        avatarState[author.id] = avatar;
-        this.setState({
-          avatars: avatarState,
+      if (author) {
+        return author.get('avatar')
+        .catch(() => {
+          return [];
+        })
+        .then((avatars) => {
+          const avatar = [].concat(avatars)[0]; // Why's this an array?
+          const avatarState = Object.assign({}, this.state.avatars);
+          avatarState[author.id] = avatar;
+          this.setState({
+            avatars: avatarState,
+          });
+          return author;
         });
-        return author;
-      });
+      } else {
+        return [];
+      }
     });
   };
 
   renderConversation = (conversation, index, allConversations) => {
     const partner = this.state.conversationPartners[conversation.id];
+    if (!partner) {
+      return (<div></div>);
```

Comment:
```suggestion
      return null;
```
I'm pretty sure JSX allows you to return null if you don't want to render an element at all.